nts.live no longer scrobbling

**Describe the bug**
When I have the nts.live tracklist open, songs are no longer scrobbled. Instead, I see the red "WS" extension icon in the extension bar.  This happens in Brave and Vivaldi.  I tried Spotify in the middle of the session to confirm that scrobbling was possible - this is why there's a Rheingold song in the logs.

**How to reproduce**
Steps to reproduce the behavior:
1. Go to https://www.nts.live/live-tracklist/the-tube (probably requires an account)

**Expected behavior**
The extension icon should change to the magnifying glass or tick icon as applicable, and the song should be scrobbled

**Debug logs**
```
main.js:1 Web Scrobbler: storage.LastFM = {
  "sessionID": "********************************",
  "sessionName": "**********"
}
main.js:1 Web Scrobbler: storage.LibreFM = null
main.js:1 Web Scrobbler: storage.ListenBrainz = {
  "sessionID": "************************************",
  "sessionName": "**********"
}
main.js:1 Web Scrobbler: storage.Maloja = null
main.js:1 Web Scrobbler: storage.Webhook = null
main.js:1 Web Scrobbler: storage.Pleroma = null
main.js:1 Web Scrobbler: storage.Options = {
  "albumGuessingAllTracks": false,
  "albumGuessingDisabled": false,
  "albumGuessingUneditedOnly": true,
  "autoToggleLove": true,
  "debugLoggingEnabled": true,
  "disabledConnectors": "[hidden]",
  "forceRecognize": false,
  "scrobbleEditedTracksOnly": false,
  "scrobblePercent": 10,
  "scrobblePodcasts": true,
  "scrobbleRecognizedTracks": true,
  "showInfobox": true,
  "useNotifications": true,
  "useUnrecognizedSongNotifications": true
}
main.js:1 Web Scrobbler: storage.Connectors = {
  "YouTube": {
    "enableGetTrackInfoFromYtMusic": false,
    "scrobbleEntertainmentOnly": false,
    "scrobbleMusicOnly": false,
    "scrobbleMusicRecognisedOnly": false
  }
}
main.js:1 Web Scrobbler: storage.ConnectorsOverrideOptions = null
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:6 getting tab id
main.js:6 undefined 1315080408
main.js:6 getting tab id
main.js:6 undefined 1315080408
main.js:6 changed mode to undefined in tab 1315080408
main.js:6 getting tab id
main.js:6 undefined 1315080408
main.js:6 getting tab id
main.js:6 undefined 1315080408
main.js:6 changed mode to undefined in tab 1315080408
main.js:6 getting tab id
main.js:6 undefined 1315080407
main.js:6 getting tab id
main.js:6 undefined 1315080407
main.js:6 changed mode to undefined in tab 1315080407
main.js:6 getting tab id
main.js:6 undefined 1315080646
main.js:6 getting tab id
main.js:6 undefined 1315080646
main.js:6 changed mode to undefined in tab 1315080646
main.js:6 changed mode to Base in tab 1315080646
main.js:6 changed mode to Base in tab 1315080646
main.js:6 changed mode to Loading in tab 1315080646
main.js:1 Web Scrobbler: Bind Last.fm scrobbler
main.js:1 Web Scrobbler: Unable to bind Libre.fm
start @ main.js:1
main.js:1 Web Scrobbler: Bind ListenBrainz scrobbler
main.js:1 Web Scrobbler: Unable to bind Maloja
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Webhook
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Pleroma
start @ main.js:1
main.js:1 Web Scrobbler: Send "get info" request: 1
main.js:1 Last.fm: track.getinfo response:
{
  "track": {
    "name": "Graffitis",
    "mbid": "c2187c5d-6b14-4fc8-99d7-0bd5691b27a5",
    "url": "https://www.last.fm/music/Rheingold/_/Graffitis",
    "duration": "221000",
    "streamable": {
      "#text": "0",
      "fulltrack": "0"
    },
    "listeners": "997",
    "playcount": "2311",
    "artist": {
      "name": "Rheingold",
      "mbid": "a21b29e1-220b-47f7-8dd3-d6d4885d807a",
      "url": "https://www.last.fm/music/Rheingold"
    },
    "album": {
      "artist": "Rheingold",
      "title": "Rheingold",
      "mbid": "5a945273-40d1-4de1-bf31-c68ee903e222",
      "url": "https://www.last.fm/music/Rheingold/Rheingold",
      "image": [
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/34s/5a0c3cc03824b74669c9b4d4049414ac.png",
          "size": "small"
        },
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/64s/5a0c3cc03824b74669c9b4d4049414ac.png",
          "size": "medium"
        },
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/174s/5a0c3cc03824b74669c9b4d4049414ac.png",
          "size": "large"
        },
        {
          "#text": "https://lastfm.freetls.fastly.net/i/u/300x300/5a0c3cc03824b74669c9b4d4049414ac.png",
          "size": "extralarge"
        }
      ],
      "@attr": {
        "position": "3"
      }
    },
    "userplaycount": "4",
    "userloved": "0",
    "toptags": {
      "tag": [
        {
          "name": "pop",
          "url": "https://www.last.fm/tag/pop"
        },
        {
          "name": "rock",
          "url": "https://www.last.fm/tag/rock"
        },
        {
          "name": "NDW",
          "url": "https://www.last.fm/tag/NDW"
        },
        {
          "name": "Neue Deutsche Welle",
          "url": "https://www.last.fm/tag/Neue+Deutsche+Welle"
        },
        {
          "name": "Das beste",
          "url": "https://www.last.fm/tag/Das+beste"
        }
      ]
    }
  }
}
main.js:6 song changed in tab 1315080646
main.js:6 Object
main.js:1 Web Scrobbler: Unable to bind Libre.fm
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Maloja
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Webhook
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Pleroma
start @ main.js:1
main.js:1 Web Scrobbler: Send "now playing" request: 2
main.js:1 Last.fm: track.updatenowplaying response:
{
  "nowplaying": {
    "artist": {
      "corrected": "0",
      "#text": "Rheingold"
    },
    "track": {
      "corrected": "0",
      "#text": "Graffitis"
    },
    "ignoredMessage": {
      "code": "0",
      "#text": ""
    },
    "albumArtist": {
      "corrected": "0",
      "#text": ""
    },
    "album": {
      "corrected": "0",
      "#text": "Rheingold"
    }
  }
}
main.js:1 ListenBrainz: {
  "status": "ok"
}
main.js:6 changed mode to Playing in tab 1315080646
main.js:6 changed mode to Paused in tab 1315080646
main.js:1 Web Scrobbler: Unable to bind Libre.fm
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Maloja
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Webhook
start @ main.js:1
main.js:1 Web Scrobbler: Unable to bind Pleroma
start @ main.js:1
main.js:1 Web Scrobbler: Send "paused" request: 2
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
Promise.catch (async)
Ys @ main.js:1
je @ main.js:1
await in je (async)
zt @ main.js:6
await in zt (async)
Ya @ main.js:6
(anonymous) @ main.js:6
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
Promise.catch (async)
Ys @ main.js:1
je @ main.js:1
await in je (async)
Qa @ main.js:6
await in Qa (async)
(anonymous) @ main.js:6
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
Promise.catch (async)
Ys @ main.js:1
je @ main.js:1
await in je (async)
zt @ main.js:6
await in zt (async)
Ya @ main.js:6
(anonymous) @ main.js:6
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
Promise.catch (async)
Ys @ main.js:1
je @ main.js:1
await in je (async)
zt @ main.js:6
await in zt (async)
Ya @ main.js:6
(anonymous) @ main.js:6
main.js:1 Error: Could not establish connection. Receiving end does not exist.
    at G (main.js:1:9173)
(anonymous) @ main.js:1
Promise.catch (async)
Ys @ main.js:1
je @ main.js:1
await in je (async)
Qa @ main.js:6
await in Qa (async)
(anonymous) @ main.js:6
```

**Environment (please complete the following information)**
-   OS: Windows 10
-   Browser: Brave, Vivaldi
-   Extension version: 3.10.0

**Additional context**
None.

--- a/src/connectors/nts-live.ts
+++ b/src/connectors/nts-live.ts
@@ -3,10 +3,10 @@ export {};
 Connector.playerSelector = '.page-live-tracks__body';
 
 Connector.artistSelector =
-	'.page-live-tracks__list li:first-child .page-live-tracks__artist-title';
+	'.page-live-tracks__list li:first-child .live-track__artist-title';
 
 Connector.trackSelector =
-	'.page-live-tracks__list li:first-child .page-live-tracks__song-title';
+	'.page-live-tracks__list li:first-child .live-track__song-title';
 
 Connector.scrobbleInfoLocationSelector = '.page-live-tracks__title';
 Connector.scrobbleInfoStyle = {

